### DISCUSSION TITLE
How do I undo a split?

### DISCUSSION THREAD

**Discussion by on3iro** [zen-browser/desktop#583] [Q&A] [answered]
I am on OSX and tried to use the shortcuts to split/unsplit. But the only way to create a split that works for me, is to select multiple tabs, right click them and chose the split action. At that point I am unable to revert this however.

Is there a way to display splits in their own respective tabs again?

Thanks in advance

**Comment by puzzle-it-nu** [comment, marked as answer]
Click the :link:  icon in the URL bar  and chose unsplit

**Reply by on3iro** [reply]
I see - thanks :)

**Reply by ilyakam** [reply]
> Click the 🔗 icon in the URL bar and chose unsplit

I couldn't find that icon in v1.15.5b on macOS. Instead, I noticed that if I put my cursor in the top-center of the split tab, these two buttons show up:

<img width="60" height="26" alt="Zen Browser Buttons to Move and Unsplit Tabs" src="https://github.com/user-attachments/assets/d0b717a0-59d7-498d-8d09-a4495115367c" />

Clicking on the double-diagonal arrow on the right unsplits the tab.

---

As for the default keyboard shortcut, it looks <kbd>⌥</kbd><kbd>⌘</kbd><kbd>U</kbd> (option + command + u) somehow conflicts with <kbd>⌘</kbd><kbd>U</kbd> (command + u) and _also_ shows the page's source code. So I changed it to <kbd>⌥</kbd><kbd>⌘</kbd><kbd>X</kbd> (option + command + x) in the keyboard settings instead. Open `about:preferences#zenCKS` in a new tab to see all keyboard settings.

**Reply by Ferrb9579** [reply]
I am unable to find that

<img width="2559" height="1526" alt="image" src="https://github.com/user-attachments/assets/2d8488f7-1509-485c-99f9-41a66aaae2e0" />

**Comment by jolo-dev** [comment]
Is there no shortcut to unsplit?

**Comment by Drake42** [comment]
I think on mac, that shortcut is taken for some reason. When I changed _break split tabs_ to something else (`Cmd + /`), it worked fine. When I tried to change it back, it was listed as **Cmd + Opt + Dead**.

You can change it by going to settings > keyboard shortcuts, scrolling down to **Split View**, and change the shortcut for Close Split View. After you enter the combo you want, hit the _escape key_, or Zen Browser won't save the new keyboard shortcut.

A number of other options for shortcuts I tried didn't work for different reasons, and ended up recorded as Cmd + Opt + (weird symbol).
